spec/v1/annotations/entity-relationship.outro.md

```diff
@@ -47,7 +47,7 @@ The ID scheme for an Entity Type ID is as following:
 <entityTypeId> := <namespace>:<entityTypeLocalId>[:v<majorVersion>]
 ```
 
-- Complete `<entityTypeId>` MUST match the regexp `^([a-z0-9-]+(?:[.][a-z0-9-]+)*):([a-zA-Z0-9._\-]+)(:(v0|v[1-9][0-9]*|))?$`
+- Complete `<entityTypeId>` MUST match the regexp `^([a-z0-9-]+(?:[.][a-z0-9-]+)*):([a-zA-Z0-9._\-]+)(:v[2-9][0-9]*|)?$`
```

Comment:
is a v10 and a v11 still allowed by that regex?

Reply:
I adjusted the regexp:

![image](https://github.com/user-attachments/assets/ea9e16e7-44a9-45cc-a063-9277fa5da9fa)